presto-main/src/main/java/io/prestosql/sql/planner/ReferenceAwareExpressionNodeInliner.java

```diff
@@ -33,7 +33,7 @@ public static Expression replaceExpression(Expression expression, Map<NodeRef<Ex
 
```

Comment:
Rephrase the commit message: "Update constructor visibility in ReferenceAwareExpressionNodeInliner"